common/src/primitives/amount.rs

```diff
@@ -15,7 +15,7 @@
 
 #![allow(clippy::eq_op)]
 
-use serialization::{Decode, Encode};
+use parity_scale_codec::{Decode, Encode};
```

Comment:
This, and similar places, should be `serialization`, not `parity_scale_codec`

Reply:
Done.